requirements/datasets.txt

```diff
@@ -16,3 +16,4 @@ bs4
 evalplus @ git+https://github.com/evalplus/evalplus@c91370f
 pyyaml
 requests
+datasets
```

Comment:
Please add it to main.txt and remove this datasets.txt, it's no longer used

Reply:
Done